src/main/java/lv/ctco/javaschool/eventorganaizer/entity/TopicDto.java

```diff
@@ -5,28 +5,66 @@
     private String topicName;
     private String topicAuthor;
     private String date;
-    private boolean isFeedbackPool;
+    private String path;
+    private Long id;
 
-    public TopicDto(String topicName, String topicAuthor, String date, boolean isFeedbackPool) {
+    public TopicDto(String topicName, String topicAuthor, String date, boolean isFeedbackPool, Long id) {
         this.topicName = topicName;
         this.topicAuthor = topicAuthor;
         this.date = date;
-        this.isFeedbackPool = isFeedbackPool;
+        if (isFeedbackPool) {
+            this.path = "pool.jsp";
+        } else {
+            this.path = "event.jsp";
```

Comment:
Please try to keep any logic outside of DTO. Pass an additional parameter to the constructor, if necessary, and move the logic to an external function.